x-pack/test/cases_api_integration/security_and_spaces/tests/common/user_actions/migrations.ts

```diff
@@ -1046,6 +1052,76 @@ export default function createGetTests({ getService }: FtrProviderContext) {
         ]);
       });
     });
+
+    describe('8.3.0', () => {
+      const CASE_ID = '5257a000-5e7d-11ec-9ee9-cd64f0b77b3c';
+
+      before(async () => {
+        await kibanaServer.importExport.load(
+          'x-pack/test/functional/fixtures/kbn_archiver/cases/8.0.0/cases.json'
+        );
+      });
+
+      after(async () => {
+        await kibanaServer.importExport.unload(
+          'x-pack/test/functional/fixtures/kbn_archiver/cases/8.0.0/cases.json'
+        );
+        await deleteAllCaseItems(es);
+      });
+
+      describe('add severity', () => {
+        it('adds the severity field to the create case user action', async () => {
+          const userActions = await getCaseUserActions({
+            supertest,
+            caseID: CASE_ID,
+          });
+
+          const createUserAction = userActions[0];
```

Comment:
It might be better to get the user action by id rather than assuming it'll aways be at index `0` right?